spec/mailers/distribution_mailer_spec.rb

```diff
@@ -27,11 +27,36 @@
     end
 
     context "with deliver_method: :pick_up" do
-      it "renders the body with 'picked up' specified" do
+      let(:mail) do
         distribution = create(:distribution, organization: @user.organization, comment: "Distribution comment", partner: @partner, delivery_method: :pick_up)
-        mail = DistributionMailer.partner_mailer(@organization, distribution, 'test subject', distribution_changes)
+        DistributionMailer.partner_mailer(@organization, distribution, 'test subject', distribution_changes)
+      end
+
+      it "renders the body with 'picked up' specified" do
         expect(mail.body.encoded).to match("picked up")
       end
+
+      context 'when parners profile pick_up_email is present' do
+        let(:pick_up_email) { 'pick_up@org.com' }
```

Comment:
We should be setting this  for non-pickup distributions as well. That way the above tests will validate that it's only going out to the partner, not the pick-up address, when it's not set to pickup.

Reply:
addressed, please take a look [813b1d9](https://github.com/rubyforgood/human-essentials/pull/4104/commits/813b1d989d1434a9bd105cf116fb7c1dc0e606f3)

Im used to write specs in slightly different style, more relying on `let` instead of instance variables. Please let me know if it does not fit well